Lib/ufo2ft/kernFeatureWriter.py

```diff
@@ -9,27 +12,34 @@ class KernFeatureWriter(AbstractFeatureWriter):
     Uses the kerning rules contained in an RFont's kerning attribute, as well as
     glyph classes from parsed OTF text. Class-based rules are set based on the
     existing rules for their key glyphs.
+
+    Uses class attributes to match UFO glyph group names and feature syntax
+    glyph class names as kerning classes, which can be overridden.
     """
 
+    leftUfoGroupRe = r"^public\.kern1\.(.+)"
+    rightUfoGroupRe = r"^public\.kern2\.(.+)"
+    leftFeaClassRe = r"@MMK_L_(.+)"
+    rightFeaClassRe = r"@MMK_R_(.+)"
```

Comment:
@jamesgk I'm not exactly sure why you differenciate public.kernX and @MMK_X names here, they both denote kerning groups. MMK groups is a widely-used ufo2 convention while public.kernX is the ufo3-mandated one. 
In both cases, they express the same just on different files.